src/odemis/acq/align/test/delphi_test.py

```diff
@@ -127,8 +133,10 @@ def test_find_lens_center(self):
         data = hdf5.read_data(os.path.join(TEST_IMAGE_PATH, "navcam-calib2.h5"))
         imgs = img.RGB2Greyscale(img.ensureYXC(data[0]))
 
-        # lens_coordinates = delphi.FindCircleCenter(data[0][0], delphi.LENS_RADIUS, 5)
-        # expected_coordinates = (-5.9703947e-05, 1.5257675e-04)  # (451.5, 445.5) px
+        # set opencv_v2 before the FindRingCenter to force unpacking findContours into 2 parameters
+        if delphi.cv2.__version__ != "3.2":
+            delphi.opencv_v2 = True
```

Comment:
It doesn't look like good idea to update `delphi.opencv_v2` here, this might interfere with other test cases. There is actually also a nicer solution, that will require you to update the code in FindRingCenter.
As explained in the link below you can adjust the `findContours` code in `FindRingCenter` to work with any version of opencv. 
https://stackoverflow.com/a/56142875

you would need to update these lines
```
    # Convert the edges into points
    if opencv_v2:
        contours, _ = cv2.findContours(edge_image, cv2.RETR_TREE, cv2.CHAIN_APPROX_NONE)
    else:
        _, contours, _ = cv2.findContours(edge_image, cv2.RETR_TREE, cv2.CHAIN_APPROX_NONE)
```
to:
```
# Convert the edges into points
contours, _ = cv2.findContours(edge_image, cv2.RETR_TREE, cv2.CHAIN_APPROX_NONE)[-2:]
```